numpy/ma/extras.py

```diff
@@ -1440,60 +1440,7 @@ class MAxisConcatenator(AxisConcatenator):
     mr_class
 
     """
-
-    def __init__(self, axis=0):
-        AxisConcatenator.__init__(self, axis, matrix=False)
-
-    def __getitem__(self, key):
-        if isinstance(key, str):
-            raise MAError("Unavailable for masked array.")
```

Comment:
Release note would be good. I assume something else uses this class and has a docstring that should be updated also. The base class itself should probably get a docstring at some point, maybe the baseclass should even be made an ABC, but that isn't needed for 1.13.